environment.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
 - scikit-learn
 - matplotlib
 - pip:
-  - mne>=0.19.1
+  - mne>=0.19.1,<0.20
```

Comment:
why this?

Reply:
that was an accidental change while moving from `~=` syntax to `>=X,<Y` syntax ... reverted in https://github.com/mne-tools/mne-bids/pull/370/commits/e9fbc57faaff30e5916d9d1694c1aa094e4c0931 Thanks!